src/Test/L0/SecretMaskerTests/SecretMaskerL0.cs

```diff
@@ -50,7 +50,7 @@ public void IsUserInfoMaskedCorrectly()
             var testSecretMasker = initSecretMasker();
 
             Assert.Equal(
-               "https://user:***@example.com",
+               "https://user:+++@example.com",
```

Comment:
>:+++@ [](http://example.com/codeflow?start=28&length=5)

here and elsewhere the use of `+++` indicates that a value matched a regex and was therefore automatically redacted. we retain `***` when redacting an explicitly specified value (e.g., a secured variable).